docker-helper

```diff
@@ -26,28 +26,56 @@ case $CMD in
     build)
         REPO="$2"
         BUILD_PATH="$3"
-        BRANCH=`echo $4 | sed 's/[\+\/]/-/g'`
-        COMMIT="$5"
-        BUILD_NUMBER="$6"
+
+
+        # Logic to determine the BRANCH tag
+        if [ -n "$4" ]; then
+            BRANCH=`echo $4 | sed 's/[\+\/]/-/g'`
+        else
+            BRANCH=`echo $(git branch | grep * | cut -d ' ' -f2) | sed 's/[\+\/]/-/g'`
+        fi
+
+        # Logic to determine the COMMIT tag
+        if [ -n "$5" ]; then
+            COMMIT="$5"
+        elif [ -n "${CIRCLE_SHA1:-}" ]; then
+            COMMIT="${CIRCLE_SHA1}"
+        elif [ -n "${TRAVIS_COMMIT:-}" ]; then
+            COMMIT="${TRAVIS_COMMIT}"
+        else
+            COMMIT=`git rev-parse --short HEAD`
```

Comment:
maybe change these to $(git rev-parse ....), i think it is less ambiguous syntax

Reply:
Were you only referring to line 34? Or Were you referring to all the if/elif statements?

Reply:
line 45, I think it is clearer to change the `` `...` `` to `$(...)`